docs/source/_topic_map.yml

```diff
@@ -5,7 +5,7 @@ Distros: openshift-*
 Topics:
   - Name: Overview
     File: index
-  - Name: Getting Started
+  - Name: Getting started
```

Comment:
I am confused. Why are we using a different standard now then the rest of docs.openshift.org. I thought we are aligning with them. AFAICT they are using capitalisation.